panda_cobot_controller:
   type: cobot_experimental_controller/PandaCobotController
   arm_id: panda
   joint_names:
       - panda_joint1
       - panda_joint2
       - panda_joint3
       - panda_joint4
       - panda_joint5
       - panda_joint6
       - panda_joint7
   collision_config:
        lower_torque_thresholds_acceleration: [25.0, 25.0, 23.0, 23.0, 20.0, 17.5, 15.0]  # [Nm]
        upper_torque_thresholds_acceleration: [25.0, 25.0, 23.0, 23.0, 20.0, 17.5, 15.0]  # [Nm]
        lower_torque_thresholds_nominal: [25.0, 25.0, 23.0, 23.0, 20.0, 17.5, 15.0] # [Nm]
        upper_torque_thresholds_nominal: [25.0, 25.0, 23.0, 23.0, 20.0, 17.5, 15.0] # [Nm]
        lower_force_thresholds_acceleration: [40.0, 40.0, 40.0, 40.0, 40.0, 40.0]  # [N, N, N, Nm, Nm, Nm]
        upper_force_thresholds_acceleration: [40.0, 40.0, 40.0, 40.0, 40.0, 40.0]  # [N, N, N, Nm, Nm, Nm]
        lower_force_thresholds_nominal:   [40.0, 40.0, 40.0, 40.0, 40.0, 40.0] # [N, N, N, Nm, Nm, Nm]
        upper_force_thresholds_nominal:   [40.0, 40.0, 40.0, 40.0, 40.0, 40.0] # [N, N, N, Nm, Nm, Nm]